gadgets/mri_core/generic_recon_gadgets/GenericImageReconGadget.cpp

```diff
@@ -101,6 +101,10 @@ namespace Gadgetron {
     {
         try
         {
+            // every image requires these two fields
```

Comment:
Seems odd that every image coming out of the Gadgetron should need those fields?

Reply:
removed. Vendor features should not be in gadgetron.